webapp/packages/plugin-data-viewer/src/locales/de.ts

```diff
@@ -51,6 +51,8 @@ export default [
   ['settings_data_editor_disable_data_copy_description', 'Deaktivieren Sie das Kopieren von Daten in Data Viewer für Nicht-Admin-Benutzer'],
   ['settings_data_editor_disable_data_export_name', 'Disable Export'],
   ['settings_data_editor_disable_data_export_description', 'Disable exporting of data in Data Viewer for non-admin users'],
+  ['settings_data_editor_disable_data_import_name', 'Disable Import'],
```

Comment:
Seems like this token is used only in plugin-data-import, I suggest to move it there